math/vecops/inc/ROOT/TVec.hxx

```diff
@@ -215,7 +216,7 @@ private:
 
 public:
    // ctors
-   TVec() = default;
```

Comment:
why?

Reply:
If the constructor is inlined, the symbol is missing in the library.

Reply:
is the ctor not inlined now that it's implicit?

Reply:
Not for all compilers, unfortunately. Some compilers still inline it. :-/
We may have to add an attribute to force not inlining. But that will affect performance for all types.
Alternatively, we may declare the constructors explicitly, along with the specializations.